Main.java

```diff
@@ -9,7 +10,24 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
+  Scanner input = new Scanner(System.in);
+
+  // ask for the integers
+  System.out.println("Please enter an integer:");
+  int a = input.nextInt();
+  System.out.println("Please enter another integer:");
+  int b = input.nextInt();
```

Comment:
Remember, variables should always be named using camelCase and describe what they are storing or being used for. a and b are probably poor names for this.